packages/data/src/components/use-select/index.js

```diff
@@ -134,6 +135,24 @@ function Store( registry, suspense ) {
 				listeningStores
 			);
 
+			if ( process.env.NODE_ENV === 'development' ) {
+				if ( ! didWarnUnstableReference ) {
+					const secondMapResult =
+						registry.__unstableMarkListeningStores(
+							() => mapSelect( select, registry ),
+							listeningStores
+						);
```

Comment:
In the second call we could skip marking the listening stores, it's redundant. A direct call:
```js
const secondMapResult = mapSelect( select, registry );
```
is all we need.